test-kitchen.gemspec

```diff
@@ -59,5 +59,7 @@ Gem::Specification.new do |gem|
   # style and complexity libraries are tightly version pinned as newer releases
   # may introduce new and undesireable style choices which would be immediately
   # enforced in CI
-  gem.add_development_dependency "finstyle",  "1.5.0"
+  #
+  # Replacing finstyle in favor of chefstyle
+  gem.add_development_dependency "chefstyle",  "0.4.0"
```

Comment:
The pattern usually is to just have chefstyle in the Gemfile, and float it on master.
https://github.com/chef/chef/blob/master/Gemfile#L77